lib/tasks/data.rake

```diff
@@ -1,8 +1,42 @@
 # frozen_string_literal: true
 
 namespace :data do
-  desc "Populates all missing ParticipantProfiles::NPQ to match NPQProfile table"
-  task populate_teacher_profiles: :environment do
-    ParticipantProfile.where(teacher_profile_id: nil).each(&:save!)
+  namespace :privacy_policy do
```

Comment:
This seems testable to me?

Reply:
➕ 
We should put the contents of the rake task in services, and test that those services behave appropriately.

Incidentally, I think we should do that with some T&P tasks that failed us at some pretty important times due to code changes breaking them :(